src/panels/config/integrations/integration-panels/ozw/ozw-node-config.ts

```diff
@@ -0,0 +1,252 @@
+                <ha-card class="content">
+                  <div class="card-content">
+                    <b>
+                      ${this._node.node_manufacturer_name}
+                      ${this._node.node_product_name} </b
+                    ><br />
+                    Node ID: ${this._node.node_id}<br />
+                    Query Stage: ${this._node.node_query_stage}
+                    ${this._metadata?.metadata.ProductManualURL
+                      ? html` <a
+                          href="${this._metadata.metadata.ProductManualURL}"
+                        >
+                          <p>Product Manual</p>
+                        </a>`
+                      : ``}
+                  </div>
+                  <div class="card-actions">
+                    <mwc-button @click=${this._refreshNodeClicked}>
+                      Refresh Node
+                    </mwc-button>
+                  </div>
+                </ha-card>
+
+                ${this._metadata?.metadata.WakeupHelp
+                  ? html`
+                      <ha-card class="content" header="Wake-up Instructions">
+                        <div class="card-content">
+                          <span class="secondary">
+                            ${this.hass.localize(
+                              "ui.panel.config.ozw.node_config.wakeup_help"
+                            )}
+                          </span>
+                          <p>
+                            ${this._metadata.metadata.WakeupHelp}
+                          </p>
+                        </div>
+                      </ha-card>
+                    `
+                  : ``}
+                ${this._config
+                  ? html`
+                      ${this._config.map(
+                        (item) => html`
+                          <ha-card class="content">
+                            <div class="card-content">
+                              <b>${item.label}</b><br />
+                              <span class="secondary">${item.help}</span>
+                              <p>${item.value}</p>
+                            </div>
+                          </ha-card>
+                        `
+                      )}
+                    `
+                  : ``}
+              `
+            : ``}
+        </ha-config-section>
+      </hass-tabs-subpage>
+    `;
+  }
+
+  private async _fetchData() {
+    if (!this.ozwInstance || !this.nodeId) {
+      return;
+    }
+
+    try {
+      this._node = await fetchOZWNodeStatus(
+        this.hass!,
+        this.ozwInstance,
+        this.nodeId
+      );
+      this._metadata = await fetchOZWNodeMetadata(
+        this.hass!,
+        this.ozwInstance,
+        this.nodeId
+      );
+      this._config = await fetchOZWNodeConfig(
+        this.hass!,
+        this.ozwInstance,
+        this.nodeId
+      );
+    } catch (err) {
+      if (err.code === ERR_NOT_FOUND) {
+        this._not_found = true;
+        return;
```

Comment:
For future proving, let's save the error code in `this._error` and render the error page with text based on that value.